Cell actions being added to more places in Security app (#3296)
elastic/security-docs: docs/getting-started/security-ui.asciidoc

[role="screenshot"]
image::images/nav-overview.gif[Overview of the navigation menu]

[float]
[[inline-actions]]
== Inline actions for fields and values

Throughout the {security-app}, you can hover over many data fields and values to display inline actions, which allow you to customize your view or investigate further based on that field or value. 

[role="screenshot"]
image::images/inline-actions-menu.png[Inline additional actions menu,45%]

In some visualizations, these actions are available in the legend by clicking a value's options icon (image:images/three-dot-icon-vertical.png[Vertical three-dot icon,16,16]).

[role="screenshot"]
image::images/inline-actions-legend.png[Actions in a visualization legend,90%]

Inline actions include the following (some actions are unavailable in some contexts):

* *Filter In*: Add a filter that includes the selected value.
* *Filter Out*: Add a filter that excludes the selected value.

> * *Add to timeline*: Launch Timeline and populate a query with the selected value.

Reviewer: We have 2 kinds of actions which add filters to the Timeline:
- **Investigate in Timeline**: Clears the Timeline, adds the cell filter(s) to timeline and automatically opens the timeline. It is used in the Detecion & Response dashboard, and in the Entity Analytics dashboard. (there are also "row actions" in the first column of the alerts/events tables that do the same thing, but those are not CellActions)

- **Add to Timeline**: Adds the cell filter(s) to Timeline without clearing the previous values, and it does not open the timeline. It is used everywhere else, alerts/events table cells, visualizations legends, details flyouts...

@stephmilovic did I miss anything?

Author: Thanks for the clarification, @semd! I think we have **Investigate in Timeline** covered correctly in other parts of the docs, and this commit corrects the docs for **Add to timeline**: https://github.com/elastic/security-docs/pull/3296/commits/13bcd00c556d82ea044c538bd6c767dcbace4707

> * *Toggle column in table*: Add or remove the selected field as a column in the alerts or events table. (This action is only available on an alert or event's details flyout.)

Reviewer 2: Should alert also be possessive here? 

```suggestion
* *Toggle column in table*: Add or remove the selected field as a column in the alerts or events table. (This action is only available on an alert's or event's details flyout.)
```

> * *Toggle column in table*: Add or remove the selected field as a column in the displayed alerts or events table. (This action is only available on an alert or event's details flyout.)

Reviewer 3: ```suggestion
* *Toggle column in table*: Add or remove the selected field as a column in a table. (This action is only available on an alert or event's details flyout.)
```
"displayed table" seemed off to me, and I think which table this option affects will be clear from context in practice

Author: Let me tinker and try to make this clearer. I'm not sure "a table" works because it's not just any table, it's the table that the user was working with (and which was displayed on screen) when they opened the details flyout.

Author: I think we can just remove "displayed." True, someone actually using the UI will probably understand the context, but someone reading this might need more specifics on which table is affected, since the whole overview page is somewhat context-less and generic:
```suggestion
* *Toggle column in table*: Add or remove the selected field as a column in the alerts or events table. (This action is only available on an alert or event's details flyout.)
```

> * *Show top _x_*: Open a modal displaying the top events or detection alerts involving the selected field.

Reviewer 4: `modal` may not be widely known, especially for non-native speakers. 
```suggestion
* *Show top _x_*: Display a pop-up window that shows the selected field's top events or detection alerts.
```

> * *Copy to Clipboard*: Copy the selected field-value pair so you can paste it elsewhere.

Reviewer 4: ```suggestion
* *Copy to Clipboard*: Copy the selected field-value pair to paste elsewhere.
```
